--- graylog.conf.orig	2023-10-24 14:55:33.695355185 +0200
+++ graylog.conf	2023-10-24 14:56:24.631974899 +0200
@@ -707,6 +707,9 @@
 # and allows administrators to control where in the file system Graylog users can select files from.
 #allowed_auxiliary_paths = /etc/graylog/data-files,/etc/custom-allowed-path
 
+# An absolute path where scripts are permitted to be executed from.
+integrations_scripts_dir = /usr/share/graylog/data/scripts
+
 # Do not perform any preflight checks when starting Graylog
 # Default: false
 #skip_preflight_checks = false
